testing/src/test/java/org/motechproject/nms/testing/it/api/UserControllerBundleIT.java

```diff
@@ -160,6 +160,16 @@ private void createKilkariTestData() {
                 sh.pregnancyPack(), SubscriptionOrigin.IVR);
     }
 
+    private void createKilkariUndeployTestData() {
```

Comment:
Since this is only used in a single method I might just inline it in that test.  Not a big deal and the change shouldn't be blocked on that.